src/looprestoration.rs

```diff
@@ -23,3 +24,448 @@ pub union LooprestorationParams {
     pub filter: Align16<[[int16_t; 8]; 2]>,
     pub sgr: LooprestorationParams_sgr,
 }
+
+type pixel = libc::c_void;
+pub type const_left_pixel_row = *const libc::c_void; // *const [pixel; 4]
+
+pub type looprestorationfilter_fn = Option<
```

Comment:
Could we just remove the `Option`?  The DSP context `struct`s aren't getting zero or default-initialized anyways, so using an `Option` isn't really affording us any extra safety, and eventually, we'll want to get rid of it anyways.  Maybe we could save this for a follow-up PR, though.

Reply:
I gave it a try and it does look like we can just remove the `Option` here. Let's do that in a separate PR though.

Reply:
Put up #333 to do this.